providers/openfeature-flagd-provider/openfeature-flagd-provider.gemspec

```
@@ -23,13 +23,13 @@ Gem::Specification.new do |spec|
   spec.files = Dir.chdir(File.expand_path(__dir__)) do
     `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   end
-  spec.bindir        = "exe"
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.bindir = "exe"
+  spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency "grpc", "~> 1.50"
 
   spec.add_development_dependency "rake", "~> 13.0"
   spec.add_development_dependency "rspec", "~> 3.12.0"
-  spec.add_development_dependency "rubocop", "~> 1.37.1"
+  spec.add_development_dependency "standard", "~> 1.35.1"
```

nitpick: I think it's safe to pessimistic lock the minor version here to get all new rule changes.

```suggestion
  spec.add_development_dependency "standard", "~> 1.35"
```